test/Common/TestHelpers.cs

```diff
@@ -85,32 +86,6 @@ internal static class TestHelpers
             {
                 DefaultAsyncRequestSleepTimeMilliseconds = httpAsyncSleepTime,
             };
-            options.NotificationUrl = notificationUrl;
-            options.ExtendedSessionsEnabled = enableExtendedSessions;
-            options.MaxConcurrentOrchestratorFunctions = 200;
-            options.MaxConcurrentActivityFunctions = 200;
-            options.NotificationHandler = eventGridNotificationHandler;
-
-            options.HubName = GetTaskHubNameFromTestName(testName, enableExtendedSessions);
-            options.Tracing = new TraceOptions()
-            {
-                TraceInputsAndOutputs = true,
-                TraceReplayEvents = traceReplayEvents,
-            };
-            options.Notifications = new NotificationOptions()
-            {
-                EventGrid = new EventGridNotificationOptions()
-                {
-                    KeySettingName = eventGridKeySettingName,
-                    TopicEndpoint = eventGridTopicEndpoint,
-                    PublishEventTypes = eventGridPublishEventTypes,
-                },
-            };
-            options.HttpSettings = new HttpOptions()
-            {
-                DefaultAsyncRequestSleepTimeMilliseconds = httpAsyncSleepTime,
-            };
-            options.NotificationUrl = notificationUrl;
```

Comment:
When i removed the `NotificationUrl` property, I noticed that there was a bunch of duplicated code here. Probably some accidental copy/paste error. I removed the duplicate lines of code.